docs/technical/security.md

```diff
@@ -0,0 +1,201 @@
+# Security Architecture
+
+The architecture for Buerokratt is almost entirely Kubernetes based.  For the MVP we will be using the Azure Kubernetes Service (AKS).

```

Comment:
I would say `The architecture for **Buerokratt**s projects DMR and CentOps is entirely Kubernetes based.`